### PR TITLE
SUIT-9281 Make sure all error messages are translated to human readable language

### PR DESCRIPTION
**Pull request by JeKachan**

### CODE REVIEW

File: lib/utils/socketErrorMessages.js [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+		return `The line is not supported by this configuration. ${toString(chainData)}`;
+	},
+	conditionNotSatisfied: ({chainData, toString}) => {
+		return `Maximum amount of key presses reached. Condition was not satisfied. ${toString(chainData)}`;
+	},
+	deviceError: ({chainData, toString, response}) => {
+		const code = responseMessageCode(response);
+		const info = responseMessageInfo(response);
+
+		if (code === 'unsupportedSelector') {
```
[the remark this answers is not in the document]
Comment (givehug) [reply lib/utils/socketErrorMessages.js]: I checked and:
- for cssSelectorInvalid, code is 'deviceFailure' on the proper branch SUIT-9291.
- When xpath is invalid IL simply returs Internal error. Which should be fixed on IL. There should be a ticket already.
- when device does not supoprt xpath code is 'unsupportedSelector'

**Comment by andriichumak** [comment]
@JeKachan I would prefer to have all texts in a single file. Can we put those to texts?

**Comment by andriichumak** [comment]
Also, please fix unit tests

**Comment by andriichumak** [comment]
@JeKachan let's commit yarn.lock only when there are changes to package.json. Please, update PR

**Pull request merged**